examples/dune

```diff
@@ -0,0 +1,9 @@
+(executables
+  (names     forward server)
+  (libraries dns-lwt-unix))
+
+(alias
+  (name    runtest)
+  (deps    forward.exe server.exe)
+; (action  (run ./forward.exe -e -v))
```

Comment:
Is the `;` commenting-out the action?

Reply:
yes, it was like that before; I guess it's because that test needs the server to run first?

Reply:
Ah fair enough, I didn't realise it was already there